src/odemis/gui/model/__init__.py

```diff
@@ -1121,6 +1121,12 @@ def __init__(self, microscope):
         # Overview streams
         self.overview_streams = model.VigilantAttribute({})  # dict: int --> stream or None
 
+        # Scintillators containing sample (manual selection in chamber tab)
+        self.active_scintillators = model.ListVA([])
+
+        # Ebeam state, 0 is off, 1 is on
+        self.ebeam_state = model.IntEnumerated(0, choices={0, 1})
```

Comment:
Use `emState`, which we already have. It has STATE_ON and STATE_OFF, (and STATE_DISABLED which could be used when the chamber is not under vacuum). That represents the e-beam *button*.

Reply:
Do we really want to disable when we are not under vacuum? Currently, the HardwareButtonController doesn't do that. I could easily add this (in a subclass of HardwareButtonController?), but I'm not sure if it's desired behaviour.